app/views/bookings/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <div class="card-booking">
+    <h2>Booking</h2>
+    <div class="card-booking-info">
+      <div>
+        <%= simple_form_for @booking do |f| %>
```

Comment:
@worksofgenius I'm sorry but I found out a bug here after merging and testing on my local host 🤦🏻‍♀️ (should have been the other way round). I reverted the merge of this PR of yours then found out you might need to recreate another one. 
As for the bug, I think it should be `<%= simple_form_for @listing, @booking do |f| %>` here? 
The error I got was 
```
NoMethodError in Bookings#new

Showing /Users/ngochoang/code/zuccamia/sharecare/app/views/bookings/new.html.erb where line #6 raised:

undefined method `bookings_path' for #<ActionView::Base:0x007f8d1091cec8>
```

Could you update this branch locally (with `git pull origin master`) and push this for a new PR? Really sorry about this.